integration_tests/integration/incidentDetailsEdit.cy.ts

```diff
@@ -272,10 +278,153 @@ context('Incident details (edit) - statement incomplete', () => {
+        firstName: 'JOHN',
+        lastName: 'SMITH',
+        assignedLivingUnit: { description: '1-2-015', agencyName: 'Moorland (HMPYOI)', agencyId: 'MDI' },
+        categoryCode: 'C',
+        alerts: [
+          { alertType: 'T', alertCode: 'TCPA' },
+          { alertType: 'X', alertCode: 'XCU' },
+        ],
+      },
+    })
+    cy.task('stubGetDraftAdjudication', {
+      id: 34,
+      response: {
+        draftAdjudication: {
+          id: 34,
+          incidentDetails: {
+            dateTimeOfIncident: '2021-10-05T13:10:00',
+            dateTimeOfDiscovery: '2021-11-05T13:10:00',
+            handoverDeadline: '2021-11-05T13:1:00',
+            locationId: 27029,
+          },
+          incidentStatement: {
+            completed: false,
+            statement: 'Statement here',
+          },
+          prisonerNumber: 'G6415GD',
+          startedByUserId: 'USER1',
+          incidentRole: {
+            associatedPrisonersNumber: 'T3356FU',
+            roleCode: '25b',
+          },
+        },
+      },
+    })
+    cy.task('stubGetLocations', {
+      agencyId: 'MDI',
+      response: [
+        {
+          locationId: 27029,
+          agencyId: 'MDI',
+          userDescription: 'Workshop 19 - Braille',
+        },
+        {
+          locationId: 27008,
+          agencyId: 'MDI',
+          userDescription: 'Workshop 2',
+        },
+        {
+          locationId: 27009,
+          agencyId: 'MDI',
+          userDescription: 'Workshop 3 - Plastics',
+        },
+        {
+          locationId: 27010,
+          agencyId: 'MDI',
+          userDescription: 'Workshop 4 - PICTA',
+        },
+      ],
+    })
+    cy.task('stubGetUserFromUsername', {
+      username: 'USER1',
+      response: {
+        activeCaseLoadId: 'MDI',
+        name: 'USER ONE',
+        username: 'USER1',
+        token: 'token-1',
+        authSource: 'auth',
+      },
+    })
+    cy.task('stubGetPrisonerDetails', {
+      prisonerNumber: 'T3356FU',
+      response: {
+        offenderNo: 'T3356FU',
+        firstName: 'JAMES',
+        lastName: 'JONES',
+        assignedLivingUnit: { description: '1-2-015', agencyName: 'Moorland (HMPYOI)', agencyId: 'MDI' },
+      },
+    })
+    cy.task('stubSearch', {
+      query: {
+        includeAliases: false,
+        prisonerIdentifier: 'T3356FU',
+        prisonIds: ['MDI'],
+      },
+      results: [
+        {
+          cellLocation: '1-2-015',
+          firstName: 'JAMES',
+          lastName: 'JONES',
+          prisonerNumber: 'T3356FU',
+          prisonName: 'HMP Moorland',
+        },
+      ],
+    })
+    cy.signIn()
+  })
+  it('should set DICOVERY button to No is DISCOVERY date same as INCIDENT date ', () => {
+    cy.visit(adjudicationUrls.incidentDetails.urls.edit('G6415GD', 34))
+    const incidentDetailsPage: IncidentDetails = Page.verifyOnPage(IncidentDetails)
+    incidentDetailsPage.radioButtonsDiscovery().find('input[value="No"]').should('be.checked')
```

Comment:
I don't think you need a whole new section for this, as date of discovery data will always be returned on the draft adjudication data on the edit page. This test could be included in the original context block if you have a new stub for stubGetDraftAdjudication (returning data with the same incident date and date of discovery) just before the test like 
`it('should set DICOVERY button to No is DISCOVERY date same as INCIDENT date ', () => {
cy.task('stubGetDraftAdjudication', {
      id: 34,
      response: {
        draftAdjudication: {
          id: 34,
          incidentDetails: {
            dateTimeOfIncident: '2021-10-05T13:10:00',
            dateTimeOfDiscovery: '2021-11-05T13:10:00',
            handoverDeadline: '2021-11-05T13:1:00',
            locationId: 27029,
          },
          incidentStatement: {
            completed: false,
            statement: 'Statement here',
          },
          prisonerNumber: 'G6415GD',
          startedByUserId: 'USER1',
          incidentRole: {
            associatedPrisonersNumber: 'T3356FU',
            roleCode: '25b',
          },
        },
      },
    })
    
    cy.visit(adjudicationUrls.incidentDetails.urls.edit('G6415GD', 34))
    const incidentDetailsPage: IncidentDetails = Page.verifyOnPage(IncidentDetails)
    incidentDetailsPage.radioButtonsDiscovery().find('input[value="No"]').should('be.checked')`

Reply:
restructured and reduced